scripts/winograd/preprocess_winograd.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+
+"""
+This file will preprocess Winograd, aligning the span indices to the tokenizaer of 
```

Comment:
'preprocess the SuperGLUE Winograd Schema Challenge data,'
There's no standard format for Winograd schemas, so need to specify.